Program.cs

```diff
@@ -1,12 +1,71 @@
-﻿using System;
+﻿using PhoneBook.StaticHelpers;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
 
 namespace PhoneBook
 {
-    class Program
+    static class Program
     {
+        private static string formatStrig = "NNNNNNNNN";
+        public static string path;
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            Console.ReadLine();
+            Runer();
+        }
+
+        private static void Runer()
+        {
+            bool flag = true;
+            while (flag)
+            {
+                Console.WriteLine("PhoneBook Program:");
+                Console.WriteLine();
+                path = @"..\\..\\..\\Samples\\FirstPhoneBook.txt";
+                string info = FileHelpers.Reader(path);
+                Console.WriteLine(info);
+                string[] sub = info.Split(' ', '\r', '\n');
+                int j = 0;
+                int count = 0;
+                List<PersonInfo> persons = new List<PersonInfo>();
+                for (int i = 0; i < sub.Length; i++)
```

Comment:
Very complex logic with i and j variables. Usually you need to split your logic between methods and call them separately. Like this: 
Get File Content
Iterate through lines
check is line valid or not
if not generate error message 
store error messages in some place or just show them